pom.xml

```diff
@@ -36,6 +36,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>1.8</java.version>
     <jmh.version>0.5.6</jmh.version>
+    <eclfw.version>7.0.0</eclfw.version>
```

Comment:
Eclipse has many frameworks but I guess this version number is unique to Eclipse Collections, right?